roles.json

```
@@ -54,6 +54,25 @@
             ]
         }
     },
+    {
+        "role": "Strategic Planner",
```

I think there is general consensus at the coordination meeting that this *must* be a committee.

```suggestion
        "role": "Strategic Planning Committee",
```

---

To be consistent with other names on the team page, I think:
```suggestion
        "role": "Strategic Planning Committee Member",
```